bootstrap/src/main/java/com/splunk/opentelemetry/javaagent/bootstrap/nocode/NocodeRules.java

```diff
@@ -28,18 +28,22 @@ public static final class Rule {
     public final String methodName;
     public final String spanName; // may be null - use default of "class.method"
     public final String spanKind; // matches the SpanKind enum, null means default to INTERNAL
+    public final String spanStatus; // may be null, should return string from StatusCodes
+
     public final Map<String, String> attributes; // key name to jexl expression
 
     public Rule(
         String className,
         String methodName,
         String spanName,
         String spanKind,
+        String spanStatus,
         Map<String, String> attributes) {
```

Comment:
I get that we're hacking here, but this constructor is now pretty unwieldy. A builder could help.